### DISCUSSION TITLE
Resize Images imageuploadeditor

### DISCUSSION THREAD

**Discussion by reach2rv** [serenity-is/Serenity#5592] [Help]
Hi,

does anyone amended imageupload editor to resize images to meet size defined? or anyone has any idea how this can be done?

**Comment by edwardch** [comment]
In your `xyzRow.cs` where you add `[ImageUploadEditor(FilenameFormat = "some/path/~")]` to your property, you should be able to pass in parameters alongside FilenameFormat that are related to width and height. ScaleHeight/ScaleWidth/etc.
